index.html

```diff
@@ -1,39 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Awesome Books</title>
-    <script src="main.js" defer></script>
-    <script defer src="book.js"></script>
-    <link rel="stylesheet" href="style.css" />
-  </head>
 
-  <body>
-    <h1>Awesome books</h1>
-    <ul id="booklist"></ul> 
-<div class="line"></div>    
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Awesome Books</title>
+  <script src="main.js" defer></script>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body onload="displayDate()">
```

Comment:
🌸 First off your code is really good and intentional. I appreciate how the website turned out on full implementation. One thing you could do is avoid adding inline JavaScript and instead target elements from your external script file. Good work overall

Reply:
I made use of inline JavaScript because I was taught about it the same day I was requested to do this project. So I feel it's okay to implement what I learned.